app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, jsonify, make_response
+from flask_sqlalchemy import SQLAlchemy
+from os import environ
+from abc import ABC, abstractmethod
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = environ.get('DATABASE_URL')
+db = SQLAlchemy(app)
+
+
+class User(db.Model):
```

Comment:
you should start organizing the structure of the app now. This belongs to db related stuff, not good to have it one file with endpoints

Reply:
will be implemented in version 2.0 when we have sorted out the other topics - I will then do a refactoring